### ISSUE TITLE
Single New Unread Email Not Bundled as Bundles

### ISSUE THREAD

**Issue by nilnocb** [leggett/simplify#632]
With all labels set to show and only priority inbox selected for viewing in settings, when received a new "Promotion" labelled email, it shows as a new, bolded un-read email with the label to the right, but it isn't shown inside a bundle. Meaning, it just looks like a regular unread email. Bundles is working correctly to move it to the top and show the label on the right, but the usual bundle heading isn't displayed. When more than one is unready, it bundles.

It would be nice to see even one new email in its bundle; helps with my mental model when visually parsing the new email counts in bundles.

Please describe the issue and the steps to reproduce it. Screenshots are useful.

---
Simplify v.3.0.0 - Configuration: aAX appMenuOpen bundleInbox ca3 caI caL cfC cfI cfO cfQ cfR cfU composeActions composeFormat darkTheme dateGroup fontMsg hideEmoji hideEmptySections hideListCount hideMsgCount hideSelectRefresh hideSignatures hideTabIcons inInbox inList invertAddons invertCompose izBgDefault listWidthLg lowDensity minimizeChat msgLabelsRight msgWidthLg nPane newUI reverseMsgs rightStar showAvatars simplify simplifyMenuOpen supExtWarn - System: Mozilla/5.0 (X11 Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36 - Window: 2256 x 1460 (200pct zoom) - Language: en

**Comment by leggett** [comment]
If there is only one message, it is not currently bundled. I found it annoying when I kept a clean inbox to have to open bundles to see / read one message. 

I plan to add settings and this could be one of them. Another crazy one I have is "don't bundle anything from today." But I also want to be careful to only add settings that a lot of people would use. I'll think about it some more... and I'd love to hear what you think after using it for a while.

**Comment by nilnocb** [comment]
Makes sense; worth a close on the issue then, unless you are using this as your backlog (and from today's 3.0 Bundle video, I suspect you are not ;-)

**Comment by leggett** [comment]
It is my secondary backlog. But yes, Workflowy is my main backlog. What I showed in the video is just one card on my kanban board.
